server_manager/model/server.ts

```diff
@@ -27,6 +27,9 @@ export interface Server {
   // Gets the version of the shadowbox binary the server is running
   getVersion(): string;
 
+  // Returns a set of experimental endpoints the server supports
+  getSupportedExperimentalEndpoints(): Promise<Set<string>>;
```

Comment:
This needs to be removed. It's not a product concept we expose. It's an implementation detail